### ISSUE TITLE
How to run falcon  in the network environment

### ISSUE THREAD

**Issue by Atbnsanw** [snwagh/falcon-public#43]
I want to test the falcon and  found 4 four IP addresses in files/IP_localhost, and I want to know the corresponding relationship between these IP addresses and PC0, PC1, and PC2, and how to change the communication port. Can the Falcon run in the network environment just by changing the IP address in files/IP_localhost?
Thanks a lot.

**Comment by snwagh** [comment]
Yes, the IPs are in order so the first one should be the IP of PC0, second one PC1 and so on. You should be able to run it over the network by simply changing the files `files/IP_*` (for instance, if you modify IP_LAN then make sure you're running the [right command/arguments](https://github.com/snwagh/falcon-public/blob/master/makefile#L6) from the `makefile`).

To change the communication port, you will have to carefully modify [this piece of the code](https://github.com/snwagh/falcon-public/blob/master/src/connect.cpp#L104-L105) to ensure that they're consistent across parties, i.e., if PC0 hosts on port X, then PC1 and PC2 need to know that.
